adapters/appnexus/appnexus.go

```diff
@@ -480,8 +504,19 @@ func NewAppNexusAdapter(config *adapters.HTTPAdapterConfig, endpoint string) *Ap
 
 func NewAppNexusBidder(client *http.Client, endpoint string) *AppNexusAdapter {
 	a := &adapters.HTTPAdapter{Client: client}
+	catmap := make(map[string]string)
+	data, err := ioutil.ReadFile("./appnexus_categorymap.json")
```

Comment:
we could add the file name through adapter config if that is an option. Let's run this by Hans and Kendall

Reply:
Should it be in category mapping data? let's say "appnexus_iab.json" format?

Reply:
That would be an option for where to store it yes. Would need to think about best way to expose it to the adapters.

Reply:
Updated this to load adapter specific options from the static folder as Hans suggested. The static/adapter-options/appnexus.json file contains the category mapping in it which gets passed along to the NewAppnexusBidder() call.